Alternative payments/Apple Pay Integration/IntegrateApplePay/Modules/ApplePay/JavaScript/ApplePay.View.js

```diff
@@ -215,7 +215,7 @@ define('DC.IntegrateApplePay.ApplePay.View', [
 			if (this.stepInfo !== undefined) {
 				_this.isReviewPage = (_this.stepInfo.step_group_name.toLowerCase() === "review") ? true : false;
 				let paymentMethodName = jQuery("[data-view='PaymentMethods.Collection']").find(".global-views-format-payment-method").html().trim();
-				_this.applepay = (paymentMethodName.toLowerCase().indexOf('applepay') !== -1) ? true : false;
+				_this.applepay = (paymentMethodName.toLowerCase().indexOf('apple') !== -1) ? true : false;
```

Comment:
Why are we changing the payment method? How or where is this defined?

Reply:
@emicha  We are identifying the Apple Pay payment method by checking if the string 'apple' exists in the payment method name. We are not changing the method name.